src/fsipy/simulations/cylinder.py

```diff
@@ -1,3 +1,223 @@
+        self.r = np.sqrt(self.A / np.pi)
+        super().__init__(**kwargs)
+
+    def update(self, t):
+        self.t = t
+        # apply a sigmoid ramp to the pressure
+        if self.t < self.t_ramp:
+            ramp_factor = -0.5 * np.cos(np.pi * self.t / self.t_ramp) + 0.5
+        else:
+            ramp_factor = 1.0
+        self.v = ramp_factor * self.v_max_final
+
+        if MPI.rank(MPI.comm_world) == 0:
+            print("v (centerline, at inlet) = {} m/s".format(self.v))
+
+    def eval(self, value, x):
+        r2 = (
+            (x[0] - self.c[0]) ** 2 + (x[1] - self.c[1]) ** 2 + (x[2] - self.c[2]) ** 2
+        )  # radius**2
+        fact_r = 1 - (r2 / self.r**2)
+
+        value[0] = -self.n[0] * (self.v) * fact_r  # *self.t
+        value[1] = -self.n[1] * (self.v) * fact_r  # *self.t
+        value[2] = -self.n[2] * (self.v) * fact_r  # *self.t
+
+    def value_shape(self):
+        return (3,)
+
+
+class InnerP(UserExpression):
+    def __init__(self, t, t_ramp, P_final, **kwargs):
+        self.t = t
+        self.t_ramp = t_ramp
+        self.P_final = P_final
+        self.P = 0.0
+        super().__init__(**kwargs)
+
+    def update(self, t):
+        self.t = t
+        # apply a sigmoid ramp to the pressure
+        if self.t < self.t_ramp:
+            ramp_factor = -0.5 * np.cos(np.pi * self.t / self.t_ramp) + 0.5
+        else:
+            ramp_factor = 1.0
+        self.P = ramp_factor * self.P_final
+
+        if MPI.rank(MPI.comm_world) == 0:
+            print("P = {} Pa".format(self.P))
+
+    def eval(self, value, x):
+        value[0] = self.P
+
+    def value_shape(self):
+        return ()
+
+
+def create_bcs(
+    dvp_,
+    DVP,
+    mesh,
+    boundaries,
+    P_final,
+    v_max_final,
+    fsi_id,
+    inlet_id,
+    inlet_outlet_s_id,
+    rigid_id,
+    psi,
+    F_solid_linear,
+    **namespace
+):
+    # Apply pressure at the fsi interface by modifying the variational form
+    p_out_bc_val = InnerP(t=0.0, t_ramp=0.1, P_final=P_final, degree=2)
+    dSS = Measure("dS", domain=mesh, subdomain_data=boundaries)
+    n = FacetNormal(mesh)
+    F_solid_linear += (
+        p_out_bc_val * inner(n("+"), psi("+")) * dSS(fsi_id)
+    )  # defined on the reference domain
+
+    # Fluid velocity BCs
+    dsi = ds(inlet_id, domain=mesh, subdomain_data=boundaries)
+    n = FacetNormal(mesh)
+    ndim = mesh.geometry().dim()
+    ni = np.array([assemble(n[i] * dsi) for i in range(ndim)])
+    n_len = np.sqrt(sum([ni[i] ** 2 for i in range(ndim)]))  # Should always be 1!?
+    normal = ni / n_len
+
+    # Parabolic Inlet Velocity Profile
+    u_inflow_exp = VelInPara(
+        t=0.0,
+        t_ramp=0.1,
+        v_max_final=v_max_final,
+        n=normal,
+        dsi=dsi,
+        mesh=mesh,
+        degree=3,
+    )
+    u_inlet = DirichletBC(DVP.sub(1), u_inflow_exp, boundaries, inlet_id)
+    u_inlet_s = DirichletBC(
+        DVP.sub(1), ((0.0, 0.0, 0.0)), boundaries, inlet_outlet_s_id
+    )
```

Comment:
If you run `flake8` again with new configuration, I think this `u_inlet_s` can be just one line. Could you consider that? Having only `)` is a bit confusing.